systest/backup/nfs-backup/docker-compose.yml

```diff
@@ -0,0 +1,377 @@
+
+version: "3.5"
+services:
+#HA backup cluster
+  alpha1:
+    image: dgraph-nfs-client
+    working_dir: /data/alpha1
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    - type: bind
+      source: ../../../tlstest/mtls_internal/tls/alpha1
+      target: /dgraph-tls
+      read_only: true
+    command: /bin/bash  /usr/local/bin/rundgraph.sh  alpha1 zero1:5080,zero2:5080,zero3:5080
+  alpha2:
+    image: dgraph-nfs-client
+    working_dir: /data/alpha2
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+    - type: bind
+      source: ../../../tlstest/mtls_internal/tls/alpha2
+      target: /dgraph-tls
+      read_only: true
+
+    command: /bin/bash  /usr/local/bin/rundgraph.sh alpha2 zero1:5080,zero2:5080,zero3:5080
```

Comment:
instead of doing this, we should ideally just use standard docker-compose files and run commands for mounting the nfs directory from the test. There may even be Go libraries that can help us here

Reply:
haven't looked at the PR completely yet, will do.Instead of docker compose, can we we use minikube instead ? so we keep can keep deployment environment and tests very similar platform.

Reply:
I don't think there is much difference in using minkube vs docker-compose. In fact, docker-compose is easier to use for running tests manually. Am I missing something here?

Reply:
so we could use production deployment file for test too

Reply:
done @aman-bansal